core/src/main/java/tech/pegasys/eth2signer/core/service/http/handlers/UpcheckHandler.java

```diff
@@ -10,18 +10,23 @@
  * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
  * specific language governing permissions and limitations under the License.
  */
-package tech.pegasys.eth2signer.core.http.handlers;
+package tech.pegasys.eth2signer.core.service.http.handlers;
 
-import static com.google.common.net.MediaType.PLAIN_TEXT_UTF_8;
 import static io.vertx.core.http.HttpHeaders.CONTENT_TYPE;
 
+import tech.pegasys.eth2signer.core.service.operations.Upcheck;
+
 import io.vertx.core.Handler;
 import io.vertx.ext.web.RoutingContext;
 
 public class UpcheckHandler implements Handler<RoutingContext> {
+  final Upcheck upcheck = new Upcheck();
 
   @Override
   public void handle(RoutingContext routingContext) {
-    routingContext.response().putHeader(CONTENT_TYPE, PLAIN_TEXT_UTF_8.toString()).end("OK");
+    routingContext
+        .response()
+        .putHeader(CONTENT_TYPE, "text/plain; charset=utf-8")
```

Comment:
Should we create our own constant if we don't want use the one from google common?

Reply:
I think so, I don't like calling toString on guava classes, we can use guava classes and create our own constants.